x/gov/keeper/deposit.go

```diff
@@ -45,7 +45,9 @@ func (keeper Keeper) DeleteAndBurnDeposits(ctx context.Context, proposalID uint6
 		return err
 	}
 
-	return keeper.bankKeeper.BurnCoins(ctx, types.ModuleName, coinsToBurn)
+	// HV2: in heimdall we have no BurnCoins func, returning nil
+	// return keeper.bankKeeper.BurnCoins(ctx, types.ModuleName, coinsToBurn)
```

Comment:
Hmmm. Is it correct to delete the deposits but neither burn not refund the coins? Will it cause some general invariant regarding total supply to no longer hold?

I have the impression it would be safer to revert this hunk, and make sure no one ever calls this method:
* either panic at the beginning of this method
* or comment it out and fix the build errors by commenting its callsites

Reply:
Yeah logically I agree. However, this function is not invoked anywhere (already removed all the callsites, compared to - for example `RefundAndDeleteDeposits` which we leverage and it's still used in `abci.go`
Also, can't re-enable the `BurnCoins` method as it's been removed from the expected `BankKeeper` (see [here](https://github.com/0xPolygon/cosmos-sdk/pull/7/files#diff-f4b8a72fd7caee61e80e763102ad4578109875ddace08521cda0efe4eadae1d3R65))
However, adding an extra `panic` at the beginning of the function sounds safe. Done [here](https://github.com/0xPolygon/cosmos-sdk/pull/15)

Reply:
👍 resolved